### PR TITLE
[Meghana Step-6] Create a Task model

### PR DESCRIPTION
**Pull request by Meghana-Reddy2**
Step 6: Let's create the task model
Create CRUD to manage tasks. First, let's create a simple configuration where only the name and details can be registered.

rails generateLet's create the model class required for task CRUD with the command
Create a migration and use it to create a table
It is important to be able to guarantee that migration can be returned to the previous state! redoGet into the habit of checking the
rails cLet's check that we can connect to the database via the model with the command
At this time, let's try creating a record with ActiveRecord.
Create a PR on GitHub and get it reviewed
If you receive a comment, please respond accordingly. If there are two LGTM (Looks Good To Me), merge them into the original branch.

Screenshot 2024-02-21 at 13 39 07
Screenshot 2024-02-21 at 13 39 36

### CODE REVIEW

File: myapp/.gitignore [hunk 1]
```diff
@@ -10,11 +10,6 @@ capybara-*.html
 rerun.txt
 pickle-email-*.html
 
-# Ignore all logfiles and tempfiles.
```
Comment (clover-zhao) [inline myapp/.gitignore:13]: In my opinion, log files and temporary files should not be included in Git.

File: myapp/app/models/task.rb
Comment (clover-zhao) [inline myapp/app/models/task.rb:1]: Do you intend to create only an empty model and add more contents at a later step?

Comment (Meghana-Reddy2) [reply myapp/app/models/task.rb:1]: Yes , I will add the data in the later steps

File: myapp/.gitignore (hunk 1 above)
Comment (Meghana-Reddy2) [reply myapp/.gitignore:13]: Yes, I mentioned them in my git ignore file. This deletion was because I wrote # Ignore all logfiles and tempfiles twice

**Review by clover-zhao — APPROVED** [review]
LGTM

**Pull request merged**